### DISCUSSION TITLE
Error "Root"

### DISCUSSION THREAD

**Discussion by Lee1887** [neuefische/questions#35] [Web] [answered]
![Bildschirmfoto 2022-08-01 um 13 27 01](https://user-images.githubusercontent.com/106076920/182138495-882c007a-4091-43a2-91e1-3376837f5612.png)

Hello, I have a small issue.
I installed everything, but it still displays a root error.
Can someone identify the issue for me by looking at the error code?
I recently switched my navbar, called sidebar, to tailwindcss, which is a little unsatisfactory.
However, it always notifies me of the problem.

Thank you very much for the response.
I'll be visiting my neuropsychiatrist in Stuttgart.

**Comment by pixelass** [comment, marked as answer]
Can you add a link to your repository/code? 
It looks like `const root` was already used before. (You cannot declare a const twice)
